ergonomics/ide.ergonomics/java.properties

```diff
@@ -22,8 +22,6 @@ projectImporter=Eclipse
 mainModule=org.netbeans.modules.java.kit
 extra.modules=org.netbeans.modules.nbjavac.*
 extra.modules.recommended.min.jdk=9
-extra.modules.1=org\\.netbeans\\.libs\\.javafx\\.(linux|win|macosx)
-extra.modules.1.recommended.max.jdk=11
```

Comment:
It is no longer necessary to download JavaFX libraries when enabling modules using them.